modules/search-layout/index.js

```diff
@@ -0,0 +1,14 @@
+import Content from './content'
+import Result from './result'
+import Results from './results'
+import Search from './search'
+import ResultStats from './result-stats'
+import Footer from './footer'
+
+Search.Content = Content
+Search.Result = Result
+Search.Results = Results
```

Comment:
This naming may be confusing. You may add an 's' using it and wonder what is wrong. That is why I usually add 'item' to it.